test/sql/include/plan_expand_hypertable_load.sql

```diff
@@ -68,6 +68,12 @@ INSERT INTO metrics_timestamptz SELECT generate_series('2000-01-01'::timestamptz
 INSERT INTO metrics_timestamptz SELECT generate_series('2000-01-01'::timestamptz,'2000-02-01'::timestamptz,'1d'::interval), 2;
 INSERT INTO metrics_timestamptz SELECT generate_series('2000-01-01'::timestamptz,'2000-02-01'::timestamptz,'1d'::interval), 3;
 
+--create a second table to test joins with
+CREATE TABLE metrics_timestamptz_2 (LIKE metrics_timestamptz);
+SELECT create_hypertable('metrics_timestamptz_2','time');
+INSERT INTO metrics_timestamptz_2
+SELECT * FROM metrics_timestamptz;
+
```

Comment:
adding a  row to metric_timestamptz and metric_timestamptz_2 which are different for both tables so that RIGHT/LEFT joins give different output.

```suggestion
INSERT INTO metrics_timestamptz_2 values( '2000-12-01'::timestamptz, 3);
```